02_configure_host.sh

```diff
@@ -130,7 +130,7 @@ if [[ "$reg_state" == "exited" ]]; then
   sudo "${CONTAINER_RUNTIME}" start registry
 elif [[ "$reg_state" != "running" ]]; then
   sudo "${CONTAINER_RUNTIME}" rm registry -f || true
-  sudo "${CONTAINER_RUNTIME}" run -d -p 5000:5000 --name registry "$DOCKER_REGISTRY_IMAGE"
+  sudo "${CONTAINER_RUNTIME}" run -d -p "${REGISTRY_PORT}:${REGISTRY_PORT}" --name registry "$DOCKER_REGISTRY_IMAGE"
```

Comment:
One of them cannot be parameterized as it is the port where the executable inthe container listens. You can only configure the exposed one

Reply:
Fixed, thanks